models/demos/wormhole/roberta/tests/test_performance.py

```diff
@@ -2,70 +2,73 @@
 
 # SPDX-License-Identifier: Apache-2.0
 
-import time
-
-import pytest
 
-from loguru import logger
+import ttnn
+import time
 import torch
+import pytest
 import transformers
 
-
-import ttnn
-
-from models.demos.bert.tt import ttnn_bert
-from models.demos.bert.tt import ttnn_optimized_bert
-
+from loguru import logger
+from models.demos.wormhole.roberta.tt import ttnn_optimized_roberta
 from ttnn.model_preprocessing import preprocess_model_parameters
 
 from models.utility_functions import (
+    is_grayskull,
     is_wormhole_b0,
-    is_blackhole,
```

Comment:
Noticing we are removing safeguards here for blackhole.
Did we confirm this passes on blackhole CI?
As far as I know, I don't think we run model tests yet

Reply:
This model is intended to run exclusively on Wormholes, so I removed unnecessary calls and added run_for_wormhole_b0 for the model tests.